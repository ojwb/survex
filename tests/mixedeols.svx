; pos=no warn=4
*fix 1 0 0 0
; This file has erratic line endings, which used to confuse cavern's line
; counting in some cases - we sprinkle in unused *fix commands to generate
; warnings so we can check the reported line numbers are what we want.

*fix 2 0 0 0


*fix 3 0 0 0


*fix 4 0 0 0
