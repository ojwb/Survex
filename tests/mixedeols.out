
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...
./mixedeols.svx:2: warning: Unused fixed point "1"
./mixedeols.svx:7: warning: Unused fixed point "2"
./mixedeols.svx:10: warning: Unused fixed point "3"

Calculating statistics...

Survey contains 3 survey stations, joined by 0 legs.
There are 0 loops.
Survey has 3 connected components.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m

There were 3 warning(s).
